shared/js/background/atb.es6.js

```diff
@@ -4,8 +4,8 @@ const utils = require('./utils.es6')
 var ATB = (() => {
     // regex to match ddg urls to add atb params to.
     // Matching subdomains, searches, and newsletter page
-    var ddgRegex = new RegExp(/^https?:\/\/(\w+\.)?duckduckgo\.com\/(\?.*|about#newsletter)/)
```

Comment:
We should avoid `new RegExp` unless the input string is dynamic/unknown. Otherwise the regex is recompiled each time the function is called.